docs/development_suite/deploy/pipeline-based/configure-gitlab-runners.md

```diff
@@ -0,0 +1,166 @@
+---
+id: configure-gitlab-runners
+title: Deploy with GitLab Runners
```

Comment:
why not GitLab CI? We generally refer to it as this, is there a difference or are they just synonyms?

Reply:
They are a sort of synonyms, but I think that if we generally refer to it as GitLab CI, it's better to use this name. I will update the PR